src/site/content/en/fast/signed-exchanges-webpackager/index.md

```diff
@@ -317,6 +317,12 @@ used with signed exchanges.
 *  You have a `CanSignHttpExchanges` certificate. This
      [page](https://github.com/google/webpackager/wiki/Certificate-Authorities)
      lists the CAs that offer this type of certificate.
+   
+*  If you don't have a certificate but have access to automated certificate renewal using
+   ACME, then you can configure your webpkgserver to automatically retrieve
```

Comment:
How does one know if they have access to automated certificate renewal using ACME? I'm assuming that's almost everybody, so maybe simplify the sentence to:

> If you don't have a certificate, then you can configure your webpkgserver to automatically retrieve certificates from your CA using ACME.

Reply:
Done.

Reply:
LGTM!